### PR TITLE
Implement ykb tnye fixes

### PR DESCRIPTION
**Pull request by Sky-High**
In the last couple of years quite some developers prepared much needed fixes for Jellyfin chromecast. @YouKnowBlom and @tnyeanderson and others compiled a list of most annoying issues in a dedicated branch https://github.com/YouKnowBlom/jellyfin-chromecast/tree/misc-fixes-tnye-merge and *almost* succeeded in releasing the fixes when a small but potentially irritating issue brought the release to a halt. See https://github.com/jellyfin/jellyfin-chromecast/pull/107#issuecomment-1402425663

The present PR is a plain merge of above branch into the unstable, with appropriate conflict resolutions, intended to release the compiled fixes and provide a clean state for moving forward with other issues. The present solution works a lot better then before, but in order to keep this PR clean no additional fixes have been included. So the above irritating issue for instance is still there. Further fixes for this and other issues will be offered in additional PR's

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
Can you rebase the pull request (again) now that #464 is merged?

File: package.json
Comment (Sky-High) [reply package.json:27]: Yes unintended. Probably caused by my installation of eslint.

**Review by nielsvanvelzen — APPROVED** [review]
Unfortunately Google screwed up my registration for the Google Cast Developer Console and their support is slow, so I'm unable to test this. The code changes look fine though and we can always fix any newly introduces bugs. The deployment is called unstable with a reason :p.

Thank you for the first PR! Looking forward to the next.

**Comment by neopc10** [comment]
At last, the unstable branch actually does something and makes the Chromecast actually useable!

We're looking forward to the next PRs that (hopefully) figure out:
1. MKV playback
2. Proper CAF queue support and ability to move forward/backward in the queue now that we can make one with more than one track.
3. [ATV Receiver library support](https://developers.google.com/cast/docs/android_tv_receiver)
4. maybe jf-vue integration??
5. maybe a custom CAF client with lyrics?

The only regression I've seen is that documentmanager with the backdrops and item details screen no longer seems to show but I believe that had something to do with one of the dependencies that was automerged over two years ago.

**Comment by nielsvanvelzen** [comment]
Thanks for your enthusiasm @neopc10! To answer some things:

> We're looking forward to the next PRs that (hopefully) figure out:

Half of this list is impossible or not related to this repository:

> MKV support

Likely not possible because browsers do not support MKV files (only WEBM which is a subset) and the cast receiver effectively runs in a browser.

> Proper CAF queue support 

Haven't read too much into CAF but this should be possible

> ATV Receiver

Unlikely to happen in the foreseeable future because I don't want a proprietary library included in the app. This prevents us from publishing to F-Droid unless we build two versions of the app which I don't want. It may also cause issues with publishing to the Amazon Appstore (I have not checked the requirements to confirm this).

> jf-vue integration

Up to the jellyfin-vue maintainers, open an issue in their repository. Good to know is that I do plan on documenting the custom messages our receiver implements.

> Lyrics support

Let's wait for Jellyfin 10.9 first 😉 

> Backdrops not working

I haven't tested the unstable version after merging this PR but I did test it yesterday and backdrops worked for me at that time. So it might be a regression or a bug that only happens in certain scenarios.

> Dependency merging

Dependencies are **not** automatically merged, I've manually merged them for a while now.

**Pull request merged**
